problemtools/tex2html.py

```diff
@@ -0,0 +1,65 @@
+#! /usr/bin/env python3
```

Comment:
This file lacks documentation and type hints. It will save us headaches later if that is added.